src/components/Accordion/components/Default.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Text } from '../../Text/Text';
+import { Box } from '../../Box/Box';
+import { Headline } from '../../Headline/Headline';
+import { Header } from './Header';
+import { ChevronUp } from './ChevronUp';
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
```

Comment:
For re-usability you can use the `Pick` typescript interface, so you don't have to repeat the same props from the `AccordionProps`.

Reply:
Done